qa/supply-chain/audits.toml

```diff
@@ -0,0 +1,116 @@
+
+# cargo-vet audits file
+
+[criteria.crypto-reviewed]
+description = "The cryptographic code in this crate has been reviewed for correctness by a member of a designated set of cryptography experts within the project."
+
+[criteria.license-reviewed]
+description = "The license of this crate has been reviewed for compatibility with its usage in this repository. If the crate is not available under the MIT license, `contrib/debian/copyright` has been updated with a corresponding copyright notice for files under `depends/*/vendored-sources/CRATE_NAME`."
+[[audits.equihash]]
```

Comment:
This probably also should have a `criteria = "crypto-reviewed"` entry.

Reply:
I was undecided on whether the code in the `equihash` crate counted as cryptographic. It definitely _uses_ cryptographic code (namely BLAKE2b).

In any case, this doesn't have to block the PR, as the `crypto-reviewed` criteria is not enforced anywhere yet (I'm still figuring out precisely how policy enforcement beyond the default of `safe-to-deploy` works, but decided we should prime the audits list with the obvious things to start with).

Reply:
It's cryptographic in my book. But as you say, it doesn't matter much yet.